microcluster/app.go

```diff
@@ -255,12 +255,45 @@ func (m *MicroCluster) RecoverFromQuorumLoss(members []cluster.LocalMember) erro
 		nodeInfo = append(nodeInfo, *info)
 	}
 
-	// Check each cluster member's /1.0 endpoint to ensure that they are unreachable
+	// Check each cluster member's /1.0 to ensure that they are unreachable.
 	// This is a sanity check to ensure that we're not reconfiguring a cluster
 	// that's still partially up.
-	// It may also be possible to check the raft term of each surviving member
-	// and redirect the user to call RecoverFromQuorumLoss on that member instead.
-	// TODO
+	remotes, err := recover.ReadTrustStore(m.FileSystem.TrustDir)
```

Comment:
Could we move these below the control socket check so we don't unnecessarily send out requests to every other cluster member if we are going to fail on a local check.